hazelcast-hibernate/hazelcast-hibernate4/src/main/java/com/hazelcast/hibernate/local/LocalRegionCache.java

```diff
@@ -197,6 +199,16 @@ public void onMessage(final Message<Object> message) {
         };
     }
 
+    @Override
+    public void memberAdded(MembershipEvent membershipEvent)
+    {
+        if (membershipEvent.getMember().getCapabilities().contains(Capability.PARTITION_HOST)) {
```

Comment:
I followed this suggestion and added `Member.hasCapability()` instead as the new public API method.